packages/k/keyutils/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : keyutils
-version    : '1.6.3'
-release    : 6
+version    : 1.6.3
+release    : 7
 source     :
     - git|https://kernel.googlesource.com/pub/scm/linux/kernel/git/dhowells/keyutils.git : v1.6.3
+homepage   : https://kernel.org/
```

Comment:
This address is too generic IMO. Let's use https://git.kernel.org/pub/scm/linux/kernel/git/dhowells/keyutils.git